plugins/maven-dependency-resolver/src/main/java/org/robolectric/internal/dependency/MavenDependencyResolver.java

```diff
@@ -146,7 +146,7 @@ private String getLocalRepositoryFromSettings(File mavenHome) {
 
       if (nodeList.getLength() != 0) {
         Node node = nodeList.item(0);
-        return node.getTextContent();
+        return node.getTextContent().trim();
```

Comment:
[The official document](https://docs.oracle.com/javase/7/docs/api/org/w3c/dom/Node.html#getTextContent()) shows `Node#getTextContent()` may return null for special node types, I can't find the spec related to `settings.xml` node type. Could you add NPE checking logic for `node.getTextContent()`? And could you add a test for it in https://github.com/robolectric/robolectric/blob/7697d1217c5eb0dee454004efe60ba219f398255/plugins/maven-dependency-resolver/src/test/java/org/robolectric/internal/dependency/MavenDependencyResolverTest.java ?

Reply:
I have some difficulty to understand how I am supposed to test that. I'll try it later when I have more time.